### PR TITLE
DS-724: FY aggregation

### PR DESCRIPTION
**Pull request by catherinedevlin**
Wraps up aggregated results to let them use the __fy transform

### CODE REVIEW

File: usaspending_api/common/mixins.py [hunk 1]
```diff
@@ -52,13 +52,31 @@ def aggregate(self, request, *args, **kwargs):
...
 
         return aggregate
 
+    _sql_function_transformations = {'fy': IntegerField}
+
+    def _wrapped_f_expression(self, col_name):
+        """F-expression of col, wrapped if needed with SQL function call
+
+        Assumes that there's an SQL function defined for each
+        registered lookup."""
```
Comment (afrasier) [inline usaspending_api/common/mixins.py]: How does this play with fk traversal? (Or is this not supported in aggregate endpoints); i.e. "recipient__location__state_code"

Comment (catherinedevlin) [inline usaspending_api/common/mixins.py]: It... does not play well, it turns out.  I took a crack at making it work, but it's not turning out to be easy.  So for now I'm adding a skipped test for it, and we can write a ticket.

Comment (afrasier) [inline usaspending_api/common/mixins.py]: (Mainly because using_the_api.md says that fk traversal works here)

Comment (catherinedevlin) [inline usaspending_api/common/mixins.py]: OK, reenabling that test.  Turns out it was just a poorly written test making it appear not to work...

**Pull request merged**